build.sbt

```diff
@@ -392,7 +392,7 @@ lazy val docs = project
       "hadoop.version" -> Dependencies.HadoopVersion,
       "extref.github.base_url" -> s"https://github.com/apache/pekko-connectors/tree/${if (isSnapshot.value) "main"
         else "v" + version.value}/%s",
-      "extref.pekko.base_url" -> s"https://pekko.apache.org/docs/pekko/current/%s",
+      "extref.pekko.base_url" -> s"https://pekko.apache.org/docs/pekko/${Dependencies.PekkoBinaryVersion}/%s",
```

Comment:
Now that I think about it more, I'm no longer sure that it's better to link to the the Pekko version this is built against than the 'current' Pekko version: because of binary compatibility it'll be common and recommended that people use newer versions of Pekko with older versions of satellite libraries.

I'm not opposed though :)

Reply:
The idea is to point to the docs of the lib based on the version of that lib that we build with. As we have seen with the Pekko core docs and the redirect of current from 1.0 to 1.1 broke some links. This approach of not linking to current means things are more stable.

Reply:
> As we have seen with the Pekko core docs and the redirect of current from 1.0 to 1.1 broke some links

I think we should avoid that in the future by leaving behind redirects, though: while changing our own sites to link to specific version indeed avoids some breakage, there might still be external links pointing to 'current'. It might also be better for SEO. Additionally, in Akka we often noticed people being confused because they were referring to outdated versions of the docs.

Having said that, I'm still OK with the change to the versioned docs - just wanted to share my thoughts for consideration.

Reply:
I've already pushed similar changes into other Pekko repos so I would like to stay consistent. We can always review the strategy again later.

The Akka docs have a feature to warn you if you are viewing old docs and provides a link to the latest docs. We could try to do something similar. One approach would be for legacy branches (eg 1.0.x branches) to be modified to put a header on all pages that provides a link to the `current` version of that page.